npm-shrinkwrap.json

```diff
@@ -389,7 +389,7 @@
           "dependencies": {
             "nan": {
               "version": "1.3.0",
-              "from": "nan@1.3.0",
+              "from": "https://registry.npmjs.org/nan/-/nan-1.3.0.tgz",
```

Comment:
npm-shrinkwrap.json  doesn't use to change from PR to PR... Are the changes correct or accidental?